o4/o4.py

```diff
@@ -955,7 +955,7 @@ def man(cmd):
             print(open_file_name, file=openf)
             if p['action'] == 'move/add':
                 move_adds.append(shquote(open_file_name))
-            if p['action'] == 'move/delete':
+            if p['action'] in ('move/delete', 'delete'):
```

Comment:
You'd think that maybe a file opened for add would have a similar problem, but it doesn't, so I didn't change line 956. Any opinion about that?